packages/fontless/docs/content/index.md

```
@@ -229,6 +229,28 @@ fontless({
 })
 ```
 
+## Global Font Faces
```

markdownlint-cli2 (0.23.2)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)